cirq/experiments/random_quantum_circuit_generation.py

```diff
@@ -154,6 +154,69 @@ def __repr__(self) -> str:
 )
 
 
+def random_rotations_between_two_qubit_circuit(
+    q0: 'cirq.Qid',
+    q1: 'cirq.Qid',
+    depth: int,
+    two_qubit_op_factory: Callable[
+        ['cirq.Qid', 'cirq.Qid', 'np.random.RandomState'], 'cirq.OP_TREE'
+    ] = lambda a, b, _: google.SYC(a, b),
```

Comment:
Btw, If you want it to be more readable, you could extract the default arguments as constants and give them a nice names.